ProcessLib/VectorMatrixAssembler.cpp

```diff
@@ -58,6 +58,42 @@ VectorMatrixAssembler::VectorMatrixAssembler(
 void VectorMatrixAssembler::assemble(
     const std::size_t mesh_item_id, LocalAssemblerInterface& local_assembler,
     const NumLib::LocalToGlobalIndexMap& dof_table, const double t,
+    const GlobalVector& x, GlobalMatrix& M, GlobalMatrix& K, GlobalVector& b)
+{
+    auto const indices = NumLib::getIndices(mesh_item_id, dof_table);
+    auto const local_x = x.get(indices);
```

Comment:
Copied from VectorMatrixAssembler::assemble and VectorMatrixAssembler::assembleWithJacobian. Now, these changes are dropped.